lib/src/entrypoint.dart

```diff
@@ -343,8 +343,7 @@ Try running `$topLevelProgram pub get` to create `$lockFilePath`.''');
     }
 
     if (!summaryOnly && hasPubspecOverrides) {
-      log.warning(
-          'Warning: pubspec.yaml has overrides from $pubspecOverridesPath');
+      log.message('pubspec.yaml has overrides from $pubspecOverridesPath');
```

Comment:
```suggestion
      log.message('Notice: pubspec.yaml has overrides from $pubspecOverridesPath');
```

Reply:
I was thinking the same, but something in the back of my head reminded me of: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

> In particular, avoid saying "Note:". It adds nothing.

Not sure that rule counts for terminal output - but somehow I think it does...

Reply:
I think it's different from source code comments.

I'm mostly thinking we should try to make it standout, so that it doesn't look like it's a continuation of the text on the previous line.

We could also write: 
```
(pubspec.yaml has overrides from $pubspecOverridesPath)
```

or 
```
hint: pubspec.yaml has overrides from $pubspecOverridesPath
```

What does it look like in normal output?

Reply:
Yeah - making it stand out is a good argument. Let's go with "Notice:".

Reply:
What we currently have is:
```
$dart pub get
Warning: pubspec.yaml has overrides from pubspec_overrides.yaml
Resolving dependencies... (1.0s)
  analyzer 0.41.2 (5.3.1 available)
  ...
  shelf_web_socket 0.2.4+1 (1.0.3 available)
! test 1.16.0 (overridden) (1.22.1 available)
  test_api 0.2.19 (0.4.17 available)
  web_socket_channel 1.2.0 (2.2.0 available)
  webkit_inspection_protocol 0.7.5 (1.2.0 available)
Warning: You are using these overridden dependencies:
! test 1.16.0 (1.22.1 available)
Got dependencies!
```

Reply:
if `pubspec_overrides.yaml` is empty:
```
$ dart pub get
Warning: pubspec.yaml has overrides from pubspec_overrides.yaml
Resolving dependencies... 
Got dependencies!
```

Reply:
If I have one override:
``` 
$ dart pub get
Warning: pubspec.yaml has overrides from pubspec_overrides.yaml
Resolving dependencies... 
! retry 3.0.0 (overridden) (3.1.0 available)
Warning: You are using these overridden dependencies:
! retry 3.0.0 (3.1.0 available)
Got dependencies!
```

Reply:
I upgrade to latest version:
```
$ dart pub get
Warning: pubspec.yaml has overrides from pubspec_overrides.yaml
Resolving dependencies... 
Warning: You are using these overridden dependencies:
! retry 3.1.0
Got dependencies!
```

Reply:
I have removed the `Warning: pubspec.yaml has overrides from pubspec_overrides.yaml` line and the `Warning: You are using these overridden dependencies:` list, and now instead always include overridden dependencies in the main update-list. And also mention if they are overridden in the pubspec_overrides.yaml file.

before:
```
> dart pub get
Warning: pubspec.yaml has overrides from pubspec_overrides.yaml
Resolving dependencies... (1.4s)
! meta 1.7.0 (overridden) (1.8.0 available)
Warning: You are using these overridden dependencies:
! meta 1.7.0 (1.8.0 available)
Got dependencies!
```
after:
```                                                                                                                                        
dart pub get
Resolving dependencies... (1.6s)
! meta 1.7.0 (overridden in ./pubspec_overrides.yaml) (1.8.0 available)
Got dependencies!
```

A lot less noisy!